x-pack/plugins/spaces/server/routes/api/public/delete.test.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+jest.mock('../../../lib/route_pre_check_license', () => {
+  return {
+    routePreCheckLicense: () => (request: any, reply: any) => reply.continue(),
```

Comment:
For these unit tests, can we make that that this is being called? We don't have a great way to do so via our api integration tests at this point, so it'd be nice to assert that we have the proper license with these.